dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/JdbcEventStore.java

```diff
@@ -97,6 +101,19 @@
 {
     private static final Log log = LogFactory.getLog( JdbcEventStore.class );
 
+    private static final String ORG_UNIT_UID_SQL = "ou.uid as ou_identifier, ";
```

Comment:
I don't think we need these constants as they are only used once and part of regular SQL. Rather have these strings inline.

Reply:
Sure. This is about personal preferences. Will change it.